# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ciphers/no_depmod.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN ciphers-v2.0f_orig/Makefile ciphers-v2.0f/Makefile
--- ciphers-v2.0f_orig/Makefile	2004-03-02 12:50:45.000000000 +0100
+++ ciphers-v2.0f/Makefile	2004-03-02 12:52:04.000000000 +0100
@@ -99,9 +99,6 @@
 modules: clean $(MLIST)
 	mkdir -p $(INSTALL_MOD_PATH)/lib/modules/$(KERNELRELEASE)/block
 	cp -p $(MLIST) $(INSTALL_MOD_PATH)/lib/modules/$(KERNELRELEASE)/block
-ifeq ($(RUNDM),y)
-	$(DEPMOD) -a $(DMOPTS)
-endif
 	sync
 	@echo "Currently running kernel is  " $(KR)
 	@echo "Modules were built for kernel" $(KERNELRELEASE)
